vulnerabilities/importers/osv.py

```diff
@@ -113,14 +128,19 @@ def get_severities(raw_data) -> Iterable[VulnerabilitySeverity]:
     """
     Yield VulnerabilitySeverity extracted from a mapping of OSV ``raw_data``
     """
-    for severity in raw_data.get("severity") or []:
-        if severity.get("type") == "CVSS_V3":
-            vector = severity["score"]
-            system = SCORING_SYSTEMS["cvssv3.1"]
-            score = system.compute(vector)
-            yield VulnerabilitySeverity(system=system, value=score, scoring_elements=vector)
-        else:
-            logger.error(f"Unsupported severity type: {severity!r} for OSV id: {raw_data['id']!r}")
+    try:
```

Comment:
Why having `try` here? What are you catching exception here for ?

Reply:
to catch a malformed CVSS3 vector 
`ERROR:vulnerabilities.importers.osv:Invalid severity Malformed CVSS3 vector, trailing "/"`
I think I should change the `Exception` to `CVSS3MalformedError`

Reply:
Yes, please do!

Reply:
Done!

Reply:
Do you have an example of `ERROR:vulnerabilities.importers.osv:Invalid severity Malformed CVSS3 vector, trailing "/"`?
(with a link)

May be we can clean them up in some cases rather than failing at parsing them?

Reply:
Yes ,  [CVSS:3.1/AV:N/AC:L/PR:L/UI:R/S:C/C:H/I:H/A:N/](https://github.com/github/advisory-database/blob/715dd424d5699294d0ce9070bc8ebc076d8abb8f/advisories/github-reviewed/2020/08/GHSA-gr4j-r575-g665/GHSA-gr4j-r575-g665.json#L14)

```
>>> from cvss import CVSS3
>>> vector = "CVSS:3.1/AV:N/AC:L/PR:L/UI:R/S:C/C:H/I:H/A:N/"
>>> c = CVSS3(vector) 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/ziad/.local/lib/python3.10/site-packages/cvss/cvss3.py", line 114, in __init__
    self.parse_vector()
  File "/home/ziad/.local/lib/python3.10/site-packages/cvss/cvss3.py", line 133, in parse_vector
    raise CVSS3MalformedError('Malformed CVSS3 vector, trailing "/"')
cvss.exceptions.CVSS3MalformedError: Malformed CVSS3 vector, trailing "/"


```
All malformed vectors : 
```
[{'type': 'CVSS_V3', 'score': 'CVSS:3.1/AV:N/AC:L/PR:L/UI:R/S:C/C:H/I:H/A:N/'}]
[{'type': 'CVSS_V3', 'score': 'CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H/'}]
[{'type': 'CVSS_V3', 'score': 'CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:N/I:N/A:H/'}]
[{'type': 'CVSS_V3', 'score': 'CVSS:3.1/AV:L/AC:H/PR:N/UI:N/S:U/C:H/I:N/A:N/'}]
[{'type': 'CVSS_V3', 'score': 'CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H/'}]
```
I think I should remove the '/' from the end of the vector to parse it successfully

Reply:
> I think I should remove the '/' from the end of the vector to parse it successfully

Go for it!

Reply:
Done!